doc/unity.rst

```diff
@@ -177,9 +176,11 @@ Scene recreation maintains continuity of physics and state in the following way:
    persisted.
 4. The MuJoCo state (for the joints that persisted) is set from the cache, and Unity transforms are synchronized.
 
-Because the MuJoCo library doesn’t (yet) expose an API for scene editing, adding and removing MuJoCo components causes
-complete scene recreation.  This can be expensive for large models or if it happens frequently.  We expect this
-performance limitation to be lifted in future versions of MuJoCo.
+MuJoCo library has functionality for dynamic scene editing (through `mjSpec
+<https://mujoco.readthedocs.io/en/stable/programming/modeledit.html#model-editing>`_), however, this is not yet
```

Comment:
two underscores please

https://stackoverflow.com/questions/27420317/restructured-text-rst-http-links-underscore-vs-use

Reply:
No, sorry. You should use an RST link here. Like this
https://github.com/google-deepmind/mujoco/blob/e9a2f055f044fa2817b41bbc6ddd23a20f58c830/doc/changelog.rst?plain=1#L30

Reply:
Thanks, pushed it